app/views/layouts/admin/application.html.erb

```diff
@@ -4,11 +4,11 @@
     <title><%= "#{yield(:title)} | " if content_for?(:title) %>PlaceCal Admin</title>
     <%= csrf_meta_tags %>
 
-    <%= stylesheet_link_tag 'admin', 'https://fonts.googleapis.com/css?family=Raleway:500,700,800|Trocchi', media: 'all', 'data-turbolinks-track': 'reload' %>
+    <%= stylesheet_link_tag 'admin', 'https://fonts.googleapis.com/css?family=Raleway:500,700,800|Trocchi', media: 'all', 'data-turbo-track': 'reload' %>
 
-    <%= javascript_pack_tag 'application', 'data-turbolinks-track': 'reload' %>
-    <%= javascript_pack_tag 'admin', 'data-turbolinks-track': 'reload' %>
-    <%= stylesheet_pack_tag 'admin', 'data-turbolinks-track': 'reload' %>
+    <%# javascript_include_tag 'application', 'data-turbo-track': 'reload', defer: true %>
```

Comment:
any rationale behind this being commented out btw?

Reply:
i changed them to exist independantly of each other, could remove this line now tbh